java/compaction-core/src/main/java/sleeper/compaction/task/CompactionTaskFinishedStatus.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.compaction.task;
+
+import sleeper.compaction.job.status.CompactionJobStatus;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+
+public class CompactionTaskFinishedStatus {
+    private final Instant finishTime;
+    private final Integer totalJobs;
+    private final double totalRuntime;
```

Comment:
Is there any reason we use an Integer (class) here but a primitive double? I think int would be better than Integer. Or do we need to allow for methods like getTotalRuntime returning null (if the task was terminated by an external action before it naturally finished then it wouldn't be able to record the time it finished).

Reply:
If totalRuntime can represent a primitive integer rounded to the nearest seconds, then we can make it a primitive int. I used double as CompactionSummary used double to store durationInSeconds

Reply:
The totalJobs variable could be an int though if thats what you are referring to